docs/reference/cluster_manifest.md

```diff
@@ -109,6 +111,12 @@ Those are parameters grouped directly under  the `spec` key in the manifest.
   is `false`, then no volume will be mounted no matter how operator was
   configured (so you can override the operator configuration).
 
+* **enableLogicalBackup**
+  Determines if the logical backup of this cluster should be taken and uploaded to S3. Default: false.
+
+* **logicalBackupSchedule**
+  Backup schedule in the Cron format. Default: "30 00 \* \* \*"
```

Comment:
It worth attaching the reference to the format:

https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#schedule

because it supports non-standard stuff, https://www.freebsd.org/cgi/man.cgi?crontab%285%29